src/backend/ai-reviewer-api/pom.xml

```diff
@@ -10,7 +10,7 @@
 	</parent>
 	<groupId>ca.bc.gov.open.jag</groupId>
 	<artifactId>ai-reviewer-api</artifactId>
-	<version>1.1.0-SNAPSHOT</version>
+	<version>1.1.1</version>
```

Comment:
Is this supposed to be 1.1.1 or 1.1.0?  If main is -SNAPSHOT, wouldn't the release be just the tagged 1.1.0?  Main would get updated to 1.1.1-SNAPSHOT.

Reply:
Wrong altogether.... I should have gone to 1.0.2 I was rushing..